eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -37,7 +37,11 @@ jobs:
       - publish: $(Build.ArtifactStagingDirectory)/Build/Products/Release-iphonesimulator
         displayName: 'Publish release artifacts [iphonesimulator]'
         artifact: 'bundles-release-iphonesimulator'
-
+      
+      - script: |
```

Comment:
I would prefer this be in its own job, that way we can run Xcode and SPM builds in parallel.